### DISCUSSION TITLE
Largar tudo e viver de freelas e outros projetos

### DISCUSSION THREAD

**Discussion by hectorgrecco** [frontendbr/forum#1131] [Discussão]
Olá, gostaria de ler histórias ou saber de experiência de alguém que já largou o emprego fixo para viver somente de freelas e/ou para tocar sua própria empresa.
Valeu =)

**Comment by renanlara** [comment]
Rapaz, já fiz isso a 2 anos atrás. A experiência é boa mas não te traz estabilidade. Fiquei um ano trabalhando remoto, voltei para o alocado onde estou a 1 ano e esse é o último mês. Vou voltar a fazer freelas.

**Comment by luizbills** [comment]
Faço freelas há 2 anos. No início é um pesadelo, toda hora procurando clientes e jobs.

No começo eu costumava olhar em sites de freelancer, mas lá era mais um leilão de quem conseguia fazer por um preço menor. Então comecei a participar de comunidades da minha área (wordpress) e tive mais sorte. Hoje em dia minha renda vem mais dos clientes fidelizados e das indicações deles.

Como o amigo ali falou, uma parte ruim é a estabilidade. Se você costuma trabalhar com sites ou sistemas, você pode até cobrar uma mensalidade para suporte e manutenção. Conseguindo muitos clientes você conseguirá uma certa estabilidade.

A parte boa é que você é obrigado a estudar educação financeira e ter um controle financeiro. Atualmente faço controle de tudo que entra e sai, sei quanto eu preciso por mês para pagar as despesas e "ser feliz" (lazer no geral). Também tenho uma porcentagem fixa da renda que eu guardo. Não só os freelancers, mas todo mundo devia guardar/investir dinheiro (você fica besta como ele cresce rápido :smile:).

Com um controle financeiro você conseguirá saber quanto você está ganhando em média e também saberá quando você ganhou mais (e também quando gastou mais :laughing:). Assim que perceber que você já está ganhando mais do que você precisa, estabeleça um **salário** para você e guarde/invista o que sobrar.

Outra dicas: 

- Não tenha medo de dizer **NÃO**. Ex: **não** faço esse tipo de serviço, **não** era esse o combinado, **não** vou conseguir entregar no prazo, etc.
- Tenha um horário fixo para trabalhar (isso inclui atender os clientes) e sempre respeite seu horário. Se possível, tenha até um lugar na casa ou num workspace onde você vai trabalhar.
- Tente centralizar o atendimento por um canal apenas. Por exemplo, eu sempre peço para o cliente mandar um e-mail com todos os detalhes.
- Aproveite os feriados e fins de semana. Não seja uma pessoa que diz: **eu não tiro feriados** ou **eu trabalho nos finais de semana também**.
- Seja um especialista. Saiba o básico de várias coisas (isso ajuda), mas seja um **mestre** em alguma coisa.
- Faça amizades com outros profissionais da sua área. Boas parcerias sempre podem surgir dessas amizades.
- Tenha metas. Faça metas de quanto você quer ganhar, quantas horas você quer trabalhar, de quantos clientes você quer ter, etc.

**Comment by hectorgrecco** [comment]
@renanlara @luizbills Muitíssimo obrigado pelas dicas! Com certeza aproveitarei todas, principalmente a de "Aproveitar feriados e fins de semana"

=)

**Comment by valterbarros** [comment]
@luizbills Boas dicas, também penso em fazer freelancer, no meu caso eu tentaria manter o trabalho e os frellas tentei em sites de freelancer também mas sem sucesso e nem recomendo esse tipo de abordagem, o foco é cliente mesmo. 

Fazer amizades na area também é uma dica boa, essa coisa de viver apenas do trabalho não é uma boa ideia inclusive isso faz parte de alguns ensinamentos do livro pai rico pai pobre.

**Comment by rubenmarcus** [comment]
Eu ja penso o contrário.
Sites de Freelancers , são bons. Eu nunca consegui nenhum, mas Upwork e Toptal, são alguns que eu recomendo.

Eu trabalhei 3 anos fazendo freelas em Wordpress para conhecidos e contatos, e pra mim foi um pesadelo. Hoje em dia vejo que se tivesse investido esses 3 anos em estudo apenas em JS e programação hoje eu estaria MUITO melhor na profissão.

Lembro que um dos meus clientes falava, "Tem uma agência aqui do Interior do RJ, que eles fazem 4 - 5 sites por mês e tiram 12mil, e vivem bem assim, porquê você não faz?"


Bem no começo eu gostava porque achava que conseguiria trabalhar e ver as séries q eu gostasse depois do trabalho, poderia acordar as 11 da manhã, ou trabalhar depois das 8 da noite até madrugada, Mas pra mim não valeu a pena.

Meus clientes não respeitavam prazos, prometiam muito, mudavam escopo toda hora, em certo mês eu recebia uma bolada, depois ficava meses sem projetos, muitos projetos tinham prazo de 1 a 2 meses, mas levaram MUITO mais que isso porque o cliente demorava demais.

Era triste ir numa festa de uma agencia que tercerizava pra mim os sites, ver a infra deles, os patrocinios e clientes, o cara abrir o laptop dele mostrar que tinha uns 15 sites pra ser feito por um valor bom, e eram sites simples institucionais, na época eu ate pensei que ia ter que empregar mais gente, mas na verdade nada disso aconteceu, o cara fazia as primeiras reuniões, passava o trabalho e sumia, o cara devendo um dinheirão pra mim, e viajando pra Europa e etc..

Cheguei ao ponto de cliente dele me ligar pra perguntar o que aconteceu.


Minha recomendação é: Tente trabalhar com clientes que são da sua área. A grande maioria dos clientes, isso em empresas e trabalhos 'normais' também. Acham que projetos são igual pastelaria, que mudar escopo é simples, e que você faz tudo em 1 dia só. 

É horrivel. E como dev, se você não é bom em arrumar clientes, e odeia essa parte de ficar lidando, você infelizmente acaba aceitando o que vem. E pra receber muitas vezes fica adiando entrega, arrumando tudo que o cliente quer. É horrivel.


Hoje 5 anos depois de ter feitos os freelas, eu mantenho apenas 2 clientes dando suporte ao site deles, mas ano que vem que acaba o suporte estou seriamente pensando, em pegar um netlify pros meus projetos e cancelar meu servidor na Bluehost, e todo meu tempo livre, fora do trabalho vai ser pra estudo / projetos pessoais, Hoje olhando pra trás , se tivesse gastado meu tempo, em um trabalho talvez as vezes ganhando um pouco menos ,mas tendo estabilidade e visão clara do crescimento, e no tempo livre, feito projetos pessoais pra praticar o conhecimento que queria adquirir, Hoje acho que ia ser um profissional bem melhor, do que ter gastado tempo com clientes que me prometiam oasis, e queriam tudo pra ontem, pagando quase nada, ou um valor que não valia o stress pelo qual eu passei.

**Comment by rubenmarcus** [comment]
Eu tinha a ilusão, que ia conseguir trabalhar de 8 a 10 meses no ano, e folgar 2 - 3 meses, conseguir viajar, horarios flexiveis, liberdade, comprometimento etc etc etc, Bom isso é possivel, se você é bom com a parte de clientes, ou tem clientes bons e é mais reconhecido no mercado, agora se você consegue clientes que não tão nem aí , que não sabem o que querem, que as vezes ficam pedindo suporte, ou marcando reunião pra coisas que você pode resolver num e-mail. Pode ter certeza que é um inferno muito maior, que trabalhar num emprego comum.

**Comment by bernardodiasc** [comment]
Pra quem gosta de desafios, tipo matar um leão por dia, esse pode ser um caminho bem interessante. Trabalhei nessa modalidade por mais de uma década, é muito puxado, e se eu puder não volto pra essa vida. Mas não me arrependo das minhas escolhas passadas, valeu a pena demais pq apesar dos problemas tem os benefícios e é por eles que entrei nessa.
